recipes/udunits2/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $(uname) != Darwin ]]; then
+    yum install -y automake libtool texinfo
```

Comment:
> The manylinux Docker image does include a number of build tools including automake but not libtool is included.

Good!  I am preparing those as packages, but we might be better with off using `manylinux`.